app/views/users/show.html.erb

```diff
@@ -0,0 +1 @@
+<h1 class="greeting">Welcome NewUser!</h1>
```

Comment:
You can make this more dynamic by using interpolation in place of 'NewUser'. Just make sure you update the feature test for this to reflect the change.

<h1 class="greeting">Welcome <%= @user.username =>!</h1>